README.md

```diff
@@ -214,19 +214,21 @@ brew install qt@5.7 mplayer
 pip3 install virtualenv
 virtualenv --python=python3 venv
 source venv/bin/activate
-pip3 install webob tornado matplotlib numpy sip pillow cheroot pyqt5 pyinstaller
+pip3 install webob tornado matplotlib numpy sip pillow cheroot decorator pyqt5==5.7.1 pyinstaller
```

Comment:
I have no idea about MacOS, but out of curiosity: why is it required to pin the pyqt5 version here?

Reply:
Packaging with pyqt5 is a total crapshoot. I pin the version of qt5 in homebrew and pyqt5 here to 5.7 to minimize churn. 

If i get a good install tonight ill consider upgrading to 5.8 or later

Reply:
I tried building with `brew install qt@5.9` and `pip3 install pyqt5==5.9 but I get an error 

    Illegal instruction: 4

I'll try again for Mnemosyne 2.7 haha